starship prompt change

GlitchDrifter opened the pull request:
Cleaner looking shell prompt. Removed the $character from the start of the prompt down to the second line.

AymanLyesri commented:
can u post a screen shot of the new style with the $character, so we could compare

GlitchDrifter commented:
![1744633334_hyprshot](https://github.com/user-attachments/assets/c2be1535-2e54-46b1-ab64-8cada327eb6a)
Here is the screenshot.

AymanLyesri commented:
No need to clutter up the prompt—just display the duration as it is, without 'last command'.

GlitchDrifter commented:
![image](https://github.com/user-attachments/assets/02ca3669-6ab7-44ac-8fd3-7d8013067684)

This is how the prompt looked before I changed it. The 'last command' was already there.

GlitchDrifter commented:
![image](https://github.com/user-attachments/assets/56f89a17-7c1d-4887-8e9c-59c5318f00bd)
I updated the prompt and removed the 'last command: ' and this is how it looks.
